src/main/java/com/converter/order/OrderConverter.java

```diff
@@ -0,0 +1,32 @@
+package com.converter.order;
+
+import com.converter.AbstractConverter;
+import com.dao.entity.client.ClientEntity;
+import com.dao.entity.order.OrderEntity;
+import com.dao.entity.product.ProductEntity;
+import com.dto.client.Client;
+import com.dto.order.Order;
+import com.dto.product.Product;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OrderConverter extends AbstractConverter<OrderEntity, Order> {
+
+    @Override
+    public Order convert(OrderEntity orderEntity) {
+        return new Order(orderEntity.getId(), orderEntity.getDateOrder(),
+                new Client(orderEntity.getClientEntity().getId(), orderEntity.getClientEntity().getName(),
```

Comment:
For ClientEntity and ProductEntity separate converters should be injected and used here